### PR TITLE
add infos with props to bigPicture & ajust css of playerList and cale…

### PR DESCRIPTION
**Pull request by Gonaline**
…ndar

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me

File: frontend/src/components/Archi/BigPicture/style.jsx [hunk 1]
```diff
@@ -1,14 +1,88 @@
...
-export default styled.div`
-  background-image: url(${Frontimg});
-  background-repeat: no-repeat;
-  background-position: 50% 15%;
+export default styled.section`
+  background-color: var(--white-opacity);
+
+  background: ${(props) => {
+    return `url(../../src/assets/Pictures/backgrounds/${props.img}.jpg) no-repeat;`;
+  }};
```
Comment (LoicBrassart) [inline frontend/src/components/Archi/BigPicture/style.jsx:8]: This code could be simplified, but it *does* work

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**